packages/strapi-admin/admin/src/components/Inputs/tests/index.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Inputs from '../index';
+
+describe('<Inputs />', () => {
+  const props = {
+    name: 'events',
+    value: ['media.create, media.delete'],
+    onChange: jest.fn(),
+  };
+
+  it('It should not crash', () => {
```

Comment:
Maybe you could generate a snapshot here and test the rendered component if the type is `headers`